SocketIoClient.h

```diff
@@ -35,7 +35,11 @@ class SocketIoClient {
 	void webSocketEvent(WStype_t type, uint8_t * payload, size_t length);
     void initialize();
 public:
+#ifdef SSL_AXTLS
     void beginSSL(const char* host, const int port = DEFAULT_PORT, const char* url = DEFAULT_URL, const char* fingerprint = DEFAULT_FINGERPRINT);
+#else
+    void beginSSL(const char* host, const int port = DEFAULT_PORT, const char* url = DEFAULT_URL, const uint8_t * fingerprint = NULL);
```

Comment:
You've changed DEFAULT_FINGERPRINT to NULL but left the definition of it in line 24